chargeio/services/PaymentMethods/PaymentMethodService.cs

```diff
@@ -1,26 +1,17 @@
 ﻿using System;
+using ChargeIo.services;
 using ChargeIO.Infrastructure;
 using ChargeIO.Models;
 
 namespace ChargeIO.Services.PaymentMethods
 {
-	public class PaymentMethodService
-	{
-        private string SecretKey { get; }
-        
-        public PaymentMethodService(string secretKey = "")
+    public sealed class PaymentMethodService : ServiceBase

```

Comment:
We shouldn't be sealing our classes -- we shouldn't be restricting users from inheriting from our classes.